frontends/web/src/routes/buy/moonpay.tsx

```diff
@@ -81,7 +81,7 @@ class Moonpay extends Component<Props, State> {
         { height }: State,
     ) {
         const account = this.getAccount();
-        if (!account) {
+        if (!account || moonpay.url === '' || moonpay.address === '') {
```

Comment:
I think just the `moonpay.url` should be enough? The address is more like an informational field. I think we could even remove it. Initially, I was planning to show the address in the UI during the onramp flow but never got to it...

Reply:
Done